### ISSUE TITLE
@Dcosthephalump's tutorial-style notebook (2 days, 16 hours)

### ISSUE THREAD

**Issue by Dcosthephalump** [nicole-brewer/jupyterlab-ide#14]
Make a tutorial notebook for this JupyterLab extension:

https://github.com/jwkvam/jupyterlab-vim

Show off JupyterLab related features and then point to the original source for everything else. (Not a LaTeX tutorial, a JupyterLab tutorial.)

## Current To-Dos
 - [x] #28
 - [ ] Create a notebook for the tutorial
   - [x] #27
     - [ ] Leave less relevant features at the end
   - [x] #31
   - [ ] Make the exercises:
     - [x] #35
     - [x] #32
     - [x] #33
 - [ ] #37
 - [ ] Commit changes
   - [x] #21
 - [ ] Clean up the notebook
   - [x] #38
   - [x] #39
   - [x] #40

[Using 15 minutes, 30 minutes, 1 hour, 2 hours, 4 hours, etc... as time blocks]

**Comment by Dcosthephalump** [comment]
It turns out that installing ```jupyterlab_latex``` requires JupyterLab >= 4.0, which is not the same as the current requirement in ```environment.yml```. Right now, that takes JupyterLab >= 3.0. Experimenting locally (though the other extensions are great!)

**Comment by nicole-brewer** [comment]
Not clear on what the attempted fix was. Did you replace the `jupyterlab>=3.0` to `jupyterlab>=4.0` in environment.yml?

**Comment by nicole-brewer** [comment]
Adding a single change to the environment often causes problems that reinstalling everything all at once can solve. I'll add this to the list of things to go over. In the meantime, feel free to get your hands dirty (and make a task list) for another extension instead!

**Comment by Dcosthephalump** [comment]
Sorry for not getting back to this earlier.

I did replace ```jupyterlab>=3.0``` with ```jupyterlab>=4.0```. After that failed, I removed the environment.

I tried reinstalling and it ended up removing a key file from Anaconda (which I verified wasn't present). The issue is documented in [other](https://github.com/jupyter/notebook/issues/5739) [places](https://github.com/jupyterlab/jupyterlab/issues/9797). After troubleshooting through a couple of the suggested fixes (uninstalling and reinstalling JupyterLab, trying to find the specific file online, and turning the computer off and on again) were not effective in my case. I uninstalled and reinstalled Anaconda. This worked!

I got started again and after looking through the current options, I think variableInspector is one that could use a tutorial. I'm still seeing if I can find an extension that would be good. Updating the above issue to reflect this shift.
